jenkins/src/cgcloud/jenkins/toil_jenkins_slave.py

```diff
@@ -56,7 +77,7 @@ def __disable_mesos_daemons( self ):
 
     @fabric_task
     def __install_parasol( self ):
-        run( "git clone git@github.com:BD2KGenomics/parasol-binaries.git" )
+        run( "git clone https://github.com/BD2KGenomics/parasol-binaries.git" )
```

Comment:
This worked for me, obviously. Otherwise I wouldn't have committed it. So the question remains why it didn't work for you. Even though I'd probably be ok with this change I don't want to mask a deeper rooted problem or misunderstanding.

Can you demonstrate that this is failing, either via shell transcript or in person?